interop/configs/branch.json

```diff
@@ -0,0 +1,40 @@
+{
+  "scripts": {
+    "base": [
+      {"action": "createGroup", "actor": "alice"},
+      {"action": "externalJoin", "actor": "alice", "joiner": "bob", "members": []},
+      {"action": "externalJoin", "actor": "alice", "joiner": "charlie", "members": ["bob"]},
+      {"action": "externalJoin", "actor": "alice", "joiner": "diana", "members": ["bob", "charlie"]},
+      {"action": "externalJoin", "actor": "alice", "joiner": "eliza", "members": ["bob", "charlie", "diana"]},
+      {"action": "branch", "actor": "alice", "members": ["bob", "charlie", "diana", "eliza"]}
+    ],
+
+    "with_extensions": [
+      {"action": "createGroup", "actor": "alice"},
+      {"action": "externalJoin", "actor": "alice", "joiner": "bob", "members": []},
+      {"action": "externalJoin", "actor": "alice", "joiner": "charlie", "members": ["bob"]},
+      {"action": "externalJoin", "actor": "alice", "joiner": "diana", "members": ["bob", "charlie"]},
+      {"action": "externalJoin", "actor": "alice", "joiner": "eliza", "members": ["bob", "charlie", "diana"]},
+      {"action": "branch", "actor": "alice", "members": ["bob", "charlie", "diana", "eliza"],
+       "extensions": [{"extension_type": 3, "extension_data": "AAAA"}, {"extension_type": 5, "extension_data": "AA=="} ]}
```

Comment:
I wonder if there's a way to check that extensions have been injected? Maybe an RPC `GetGCEs` where the actor returns `repeated Extension extensions`? I'm worried that implementation 1 can't generate GCEs and creates a branch without them, which cant be noticed by other implementations

Reply:
https://github.com/mlswg/mls-implementations/issues/154